docs/interpreter/springxd.md

```diff
@@ -0,0 +1,150 @@
+  </tr>  
+</table>
+
+> Note: A single Zeppelin `paragraph` can contain multiple `streams` or multiple `jobs`, but it can not mix-up jobs and streams in the same paragraph!
+
+<br/>
+[<img align="right" src="http://img.youtube.com/vi/uSUBe8Hgk8w/0.jpg" alt="zeppelin-view" hspace="10" width="250"></img>](https://www.youtube.com/watch?v=uSUBe8Hgk8w)
+
+Following [video tutorial](https://www.youtube.com/watch?v=uSUBe8Hgk8w) illustrates some of the features provided by the `SpringXD Interpreter`. It shows how to build SpringXD pipeline to ingest a real-time Tweet stream into Geode, HDFS and HAWQ backends. 
+
+### Create Interpreter
+
+By default Zeppelin creates one `SpringXD` interpreter instance that communicates with the backend SpringXD system
+
+To create new SpringXD instance open the `Interpreter` section and click the `+Create` button. Pick a `Name` of your choice and from the `Interpreter` drop-down select `xd`. Then follow the configuration instructions and `Save` the new instance.
+
+> Note: The `Name` of the instance is used only to distinct the instances while binding them to the `Notebook`. The `Name` is irrelevant inside the `Notebook`. In the `Notebook` you must use `%xd.stream` or `%xd.job` tag.
+
+### Bind to Notebook
+In the `Notebook` click on the `settings` icon in the top right corner. The select/deselect the XD interpreter to be bound with the `Notebook`.
+
+### Configuration
+You can modify the configuration of the SpringXD from the `Interpreter` section. The SpringXD interpreter express the following properties:
+
+ <table class="table-configuration">
+   <tr>
+     <th>Property Name</th>
+     <th>Description</th>
+     <th>Default Value</th>
+   </tr>
+   <tr>
+     <td>springxd.url</td>
+     <td>The URL for SpringXD REST API. </td>
+     <td>http://localhost:9393</td>
+   </tr>
+ </table>
+
+### How to use
+Following sections will explain how to create and destroy Streams and Jobs inside the Zeppelin paragraphs.
+> Tip: Use `CTRL + .` for SpringXD auto-completion.
+
+
+#### XD Streams
+
+* Start the paragraphs with the full `%xd.stream` prefix tag to identifie the SpringXD Streams interpreter.
+* Use `Ctrl+.` for auto-completion.
+* In the paragraph define one or more stream definitions - each on a separate line.
+* Every stream definition must have name. Follow the convention: `stream name = stream definition`. Streams without names are ignored. The stream definition follows the Spring XD DSL: [DSL Guide](http://docs.spring.io/spring-xd/docs/current/reference/html/#dsl-guide) 
+* Every time a Paragraph is Run it destroys any previous streams created in the paragraph. The previous streams are destroyed even if their names have been changed or removed.
+* Paragraph `Run` command will `Create` and automatically `Deploy` all streams defined in the Paragraph.
+* If one stream fails to create or to deploy then all streams in the paragraph are destroyed
```

Comment:
Since all of the other sentences have dots, so why don't you add a `dot (.)` in the end of this sentence too ?